### ISSUE TITLE
This Solution isn't upto date 

### ISSUE THREAD

**Issue by PSI-KID-Mercy** [amanchadha/coursera-deep-learning-specialization#27]
This Solution isn't upto date  as i looked up into the matter, the updated version requires more packages to be initiated 

import numpy as np
import copy
import matplotlib.pyplot as plt
import h5py
import scipy
from PIL import Image
from scipy import ndimage
from lr_utils import load_dataset
from public_tests import *

%matplotlib inline
%load_ext autoreload
%autoreload 2

*and so with this we will have different solution*

**Comment by BHUVAN-RJ** [comment]
yes , I have created a pull request about that i hope he accepts
